client/src/pages/ExpandedMentee.jsx

```diff
@@ -0,0 +1,347 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { ref, uploadBytes, getDownloadURL } from 'firebase/storage';
+import PropTypes from 'prop-types';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import styles from '../styles/Mentees.module.css';
+import VideoIcon from '../assets/icons/videos_icon.svg';
+import ImageIcon from '../assets/icons/images_icon.svg';
+import FlyerIcon from '../assets/icons/flyers_icon.svg';
+import { storage } from './firebase';
```

Comment:
Can you change this to an api call just so we have less to do when we clean up?

Reply:
should i move the entire "addMedia" function to the server, or just this part:
`const storageRef = ref(storage, `/images/${fileName}`);
      uploadBytes(storageRef, files).then((snapshot) => {
        getDownloadURL(snapshot.ref).then((url)` ?

cause the addMedia function sets a lotta states on the expanded mentee & media pages.